pipenv/environments.py

```diff
@@ -26,7 +26,7 @@
 PIPENV_DEFAULT_THREE = os.environ.get('PIPENV_DEFAULT_THREE')
 
 # Tells Pipenv to use the virtualenv-provided pip instead.
-PIPENV_USE_SYSTEM = os.environ.get('VIRTUAL_ENV') if not 'PIPENV_IGNORE_VIRTUALENVS' in os.environ else False
+PIPENV_USE_SYSTEM = os.environ.get('VIRTUAL_ENV') if 'PIPENV_IGNORE_VIRTUALENVS' not in os.environ else False
```

Comment:
This is a good spot! I think it should be submitted separate from this PR though for the same reasons listed above.